netsuite/10.0/modules/ROOT/pages/netsuite-reference.adoc

```diff
@@ -990,7 +990,7 @@ Used to change the email address for the account.
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| New Email a| String |  New email value. |  `#[payload]` |
+| New Email a| String |  New email value. |  |
 | Current Credentials a| String |  current user password. |  | x
 | Just This Account a| Boolean |  Update email for the current account only. |  true |
```

Comment:
```suggestion
| Just This Account a| Boolean |  Update email for the current account only |  true |
```
Is "update email" a modified noun or is this a verb? If the latter, can we do something to avoid inconsistency?